<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftUpdatePromptWindow</class>
 <widget class="QDialog" name="SoftUpdatePromptWindow">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>191</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>16777210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New VoxOx Release</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <item row="0" column="0" rowspan="7">
    <widget class="QLabel" name="lblIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>105</width>
       <height>103</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>105</width>
       <height>103</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>voxox.css</string>
     </property>
     <property name="pixmap">
      <pixmap>:/pics/auto_updater_recommended.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="4">
    <widget class="QLabel" name="lblPrompt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777210</height>
      </size>
     </property>
     <property name="text">
      <string>A new version of VoxOx (2.0.xx) is available.  &lt;p&gt;Would you like to install it now?&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>205</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" colspan="4">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Do not ask me again</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="4">
    <widget class="QLabel" name="lblDownloading">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Downloading:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>0</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="installButton">
     <property name="text">
      <string>Install Now</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
  <include location="../../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
